src/app/page.tsx

```diff
@@ -1,95 +1,21 @@
 import Image from "next/image";
```

Comment:
In order to use properly the `onChange` function we should change this component to a "client" component, let's add this line `"use client";` at the top of this file.

For more information read https://nextjs.org/docs/app/building-your-application/rendering/client-components

Reply:
👍